templates/territorial_council/political_committee/partials/_sidebar.html.twig

```diff
@@ -0,0 +1,23 @@
+<aside class="territorial-council__aside l__wrapper--aside">
+    {% set territorial_council = political_committee.territorialCouncil %}
+    {% set president = territorial_council.memberships.president %}
+    {% if president %}
+        {% set president = president.adherent %}
+
+        <div id="territorial-council-referent" class="territorial-council__aside--section">
+            <h5>Président du Comité politique</h5>
+
+            <div class="territorial-council__host l__row l__row--top text--body b__nudge--bottom">
+                {% include 'territorial_council/partials/_member_identity_box.html.twig' with {
+                    initials: president.initials,
+                    full_name: president.fullName,
+                    label: 'Référent(e)'
+                } %}
+
+                <a href="{{ path('app_adherent_contact', {uuid: president.uuid, from: 'territorial_council', id: territorial_council.uuid.toString}) }}" class="link--no-decor text--blue--dark text--small m-auto--left">
```

Comment:
should we redirect the user on the political committee page or on coterr page? 
maybe change `from` attribute by `political_committee`?